doc/reference/usermode/syscalls.rst

```diff
@@ -188,7 +190,7 @@ may or may not be used.
    System Call execution flow
 
 Some system calls may have more than six arguments. The number of
-arguments passed via registers is limited to six for all
+32-bit arguments passed via registers is limited to six for all
```

Comment:
omit this, but you could say "arguments passed directly via registers" i.e. no splitting or ancillary data structures

Reply:
Done.